include/sealib/eulertrail/recursivedyckmatchingstructure.h

```diff
@@ -1,9 +1,9 @@
-#ifndef SEALIB_RECURSIVEDYCKMATCHINGSTRUCTURE_H_
-#define SEALIB_RECURSIVEDYCKMATCHINGSTRUCTURE_H_
+#ifndef SEALIB_EULERTRAIL_RECURSIVEDYCKMATCHINGSTRUCTURE_H_
+#define SEALIB_EULERTRAIL_RECURSIVEDYCKMATCHINGSTRUCTURE_H_
 
-#include <sealib/localdycktable.h>
-#include <sealib/dyckmatchingstructure.h>
-#include <sealib/dictionary/rankselect.h>
+#include "../../../src/eulertrail/localdycktable.h"
```

Comment:
You can't reference the *src* folder from a public header file. Either make this file private or the other one public.